config/environments/production.rb

```diff
@@ -88,4 +88,19 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  # Setup the mailer config
+  config.action_mailer.default_url_options = {host: ENV['HOST']}
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.smtp_settings = {
+    :user_name => ENV['SENDGRID_USERNAME'],
+    :password => ENV['SENDGRID_PASSWORD'],
+    :domain => 'https://mr-clean.herokuapp.com',
```

Comment:
Can you get away with just providing the `ENV['HOST']` here? Or, are there any other environment variables that work?